packages/ui-color-picker/src/ColorMixer/ColorPalette/index.tsx

```diff
@@ -102,17 +113,18 @@ class ColorPalette extends Component<ColorPaletteProps, ColorPaletteState> {
     return this._paletteRef!.getBoundingClientRect().height
   }
   calcSaturation = (position: number) =>
-    Math.round((position / this.paletteWidth) * 100) / 100
+    Math.round(((position - this.padding) / this.movableWidth) * 100) / 100
```

Comment:
Now the movable area of the indicator is smaller, we need to adjust the color calculation formula